language/functional_tests/tests/testsuite/epilogue/increment_sequence_number_on_tx_script_failure.mvir

```diff
@@ -0,0 +1,20 @@
+//! account: default, 1000000, 0
+main() {
+    assert(false, 77);
+    return;
+}
+
+
+
+//! new-transaction
+import 0x0.LibraAccount;
+
+main() {
+    assert(get_txn_sequence_number() == 1, 42);
+    assert(LibraAccount.sequence_number(get_txn_sender()) == 1, 42);
+    return;
+}
+
+// check: ABORTED
```

Comment:
Does it also work do add `check: ABORTED` and `check: 77` after the first transaction? That seems easier to read. Otherwise, it's a bit hard to tell which checks are associated with the first transaction and which are associated with the second.